test/bddtests/fixtures/.env

```diff
@@ -0,0 +1,18 @@
+#
+# Copyright SecureKey Technologies Inc. All Rights Reserved.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+# This file contains the default images and tags used in the docker-based
+# Fabric fixtures. The images and tags can be overridden using environment
```

Comment:
There should be no references to Fabric in this project.

Reply:
done